x-pack/test/security_solution_endpoint_api_int/apis/endpoint_response_actions/agent_type_support.ts

```diff
@@ -15,9 +15,8 @@ export default function ({ getService }: FtrProviderContext) {
   describe('Response Actions support for sentinelOne agentType', function () {
     targetTags(this, ['@ess', '@serverless']);
 
-    describe('and the "responseActionsSentinelOneV1Enabled" feature flag is disabled', () => {
-      // When feature flag is enabled, this entire `describe()` block should be removed
-      it('should return an error', async () => {
+    describe('and the "responseActionsSentinelOneV1Enabled" feature flag is enabled', () => {
```

Comment:
This is fine, but we really should have tests for a) feature flag `true` and b) feature flag `false`

Reply:
Thank you Paul, you're right, I'll fix the test to include both flags. But I'll do that in a new PR tomorrow, to just get this one out of the way :)